pkg/vulnsrc/mariner/mariner.go

```diff
@@ -206,8 +206,11 @@ func (vs VulnSrc) commit(tx *bolt.Tx, platformName string, entries []Entry) erro
 	for _, entry := range entries {
 		cveID := entry.Metadata.Reference.RefID
 		advisory := types.Advisory{}
-		if entry.Metadata.Patchable {
+		// Definition.Metadata.Patchable has a bool and "Not Applicable" string.
+		if entry.Metadata.Patchable == "true" {
```

Comment:
We may want to care about the upper letters.

```suggestion
		if strings.ToLower(entry.Metadata.Patchable) == "true" {
```